packages/block-library/src/post-tags/index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-tags` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-tags` block on the server.
+ *
+ * @return string Returns the filtered post tags for the current post wrapped inside "a" tags.
+ */
+function render_block_core_post_tags() {
+	$post = gutenberg_get_post_from_context();
+	if ( ! $post ) {
+		return '';
+	}
+	$post_tags = get_the_tags();
+	if ( ! empty( $post_tags ) ) {
+		$output = '';
+		foreach ( $post_tags as $tag ) {
+			$output .= '<a href="' . get_tag_link( $tag->term_id ) . '">' . $tag->name . '</a>' . ' | ';
```

Comment:
We should get to a point where we support the common customization the theme function tags offer: https://codex.wordpress.org/Function_Reference/the_tags